charts/merlin/templates/merlin-deployment.yaml

```diff
@@ -174,6 +174,12 @@ spec:
             value: {{ .Values.merlin.transformer.feast.bigtableCredential}}
           - name: DEFAULT_FEAST_SOURCE
             value: {{ .Values.merlin.transformer.feast.defaultFeastSource}}
+          - name: FEAST_SERVING_KEEP_ALIVE_ENABLED
```

Comment:
Can you update merlin chart in https://github.com/caraml-dev/helm-charts/tree/main/chartshttps://github.com/caraml-dev/helm-charts/tree/main/charts